packages/shared/lib/time.ts

```diff
@@ -0,0 +1,45 @@
+import { get, writable } from 'svelte/store'
+
+export const MILLISECONDS_PER_SECOND = 1000
```

Comment:
I think that these are global constants to be used throughout the code, so I don't think that they should be in the same file as the Idle util functions.

Reply:
Hmm then I think the constants should stay where they are then, and I'll put the idle code back in the component (as that's the only place it's used anyway). Although, I think using `time-constants.ts` isn't the way either, since that creates an inconsistency with the rest of the code in `shared/lib`. 

After some type of restructure though, would be good to look at separating constants from logic (along with re-organizing enums, interfaces, functions, etc.).

Reply:
Agree on the refactor. What inconsistencies are created?